Add section handy command when development HiGlass (#16)
higlass/higlass-manage: README.md

higlass-manage stop
```

## Development

> The following is a list of handy command then developing HiGlass:

Reviewer: ```suggestion
The following is a list of handy commands when developing HiGlass:
```

> - **Start locally build docker image**:

Reviewer: ```suggestion
- **Start locally built docker image**:
```

> 

Reviewer: ```suggestion
The locally built image must be named `image-default`. Usually built using [higlass-docker](https://github.com/higlass/higlass-docker/). 
```
